src/services/auth.service.ts

```diff
@@ -29,7 +29,19 @@ export const registerUser = async (
   })
   await profileRepository.save(newProfile)
 
-  return newProfile
```

Comment:
Why can we return newProfile? It's not returning the password right?

Reply:
![https://imgur.com/hkwl7H2.png](https://imgur.com/hkwl7H2.png)
this is the result when using newProfile even when I set password select option to false